src/Avalonia.Base/AvaloniaPropertyMetadata.cs

```diff
@@ -8,13 +8,15 @@ namespace Avalonia
     public class AvaloniaPropertyMetadata
     {
         private BindingMode _defaultBindingMode;
+        private UpdateSourceTrigger _updateSourceTrigger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AvaloniaPropertyMetadata"/> class.
         /// </summary>
         /// <param name="defaultBindingMode">The default binding mode.</param>
         public AvaloniaPropertyMetadata(
-            BindingMode defaultBindingMode = BindingMode.Default)
+            BindingMode defaultBindingMode = BindingMode.Default,
+            UpdateSourceTrigger updateSourceTrigger = UpdateSourceTrigger.Default)
```

Comment:
Shouldn't this constructor be setting the field value? The `updateSourceTrigger` parameter seems to be unused.